sources/packages/backend/apps/db-migrations/src/sql/CASInvoiceBatches/Create-cas-invoice-batches.sql

```diff
@@ -0,0 +1,37 @@
+CREATE TABLE sims.cas_invoice_batches(
+  id SERIAL PRIMARY KEY,
+  batch_name VARCHAR(50) NOT NULL,
+  batch_date DATE NOT NULL,
```

Comment:
Minor. Technical AC mentions batch date as timestamp. But date sounds good to me.

Reply:
Yes, there was some back and forth about where it would be used.
I will double-check, if it will be used only by SIMS I will revert to timestamp.

Reply:
Converted to timestamp.